extras/jsonrpc/daemon_types.go

```diff
@@ -321,6 +317,20 @@ type Claim struct {
 	ValidAtHeight           int              `json:"valid_at_height,omitempty"`
 }
 
+type Meta struct {
+	ActivationHeight  int64   `json:"activation_height,omitempty"`
+	CreationHeight    int64   `json:"creation_height,omitempty"`
+	CreationTimestamp int     `json:"creation_timestamp,omitempty"`
```

Comment:
int? are you sure?

Reply:
I based it off of a different timestamp elsewhere. I checked and you had added that timestamp. So the real question is..."are you sure?". #ITrustNiko

Reply:
that value comes directly from the blockchain header file, it's a 32 bit unsigned integer:
https://en.bitcoin.it/wiki/Protocol_documentation#Block_Headers

Reply:
okay, the default is 32bit in go so we're good. Not unsigned but not a big deal.